src/admin/paths/admin@reports@remove_approvers.yaml

```diff
@@ -0,0 +1,46 @@
+post:
+  tags:
+    - Reports
+  summary: Remove Approvers
+  description: |
+    Remove Approvers
+  operationId: report_remove_approvers
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              $ref: ../../components/schemas/report.yaml#/remove_approvers_in
+  responses:
+    '200':
```

Comment:
remove approver is not bulk 
so why the response is simple 200
it should ideally be report object